src/fastoad_cs25/models/variable_descriptions.txt

```diff
@@ -318,6 +318,8 @@ settings:weight:airframe:fuselage:mass:k_lg || correction coefficient: 1.05 if m
 settings:weight:airframe:landing_gear:front:weight_ratio || part of aircraft weight that is supported by front landing gear
 settings:weight:airframe:wing:mass:k_mvo || 1.39 for Airbus type aircrafts
 settings:weight:systems:power:mass:k_elec || electricity coefficient: 1.00 if 2 engines (A300, A310 type) / (1.02 if 2 engines (DC9, Caravelle type) / 1.03 if 3 engines (B727 type) / 1.05 if 3 engines (DC10, L1011 type) / 1.08 if 4 engines (B747 type)
+settings:geometry:horizontal_tail:area_factor || scaling factor applied to horizontal tail area
+settings:geometry:vertical_tail:area_factor || scaling factor applied to vertical tail area
```

Comment:
To move into "tuning" variables in addition to rename

Reply:
Corrected in [4b80055](https://github.com/fast-aircraft-design/FAST-OAD_CS25/pull/49/commits/4b800553e8258931efb928e82aa882275b69fe91)